pybikes/data/nextbike.json

```diff
@@ -2051,19 +1227,7 @@
             }
         },
         {
-            "domain": "mb",
```

Comment:
The feed for Budapest doesn't need to bee deleted, it has been fixed with a new domain here: #459